### ISSUE TITLE
Status Page fails to update correctly

### ISSUE THREAD

**Issue by Deathbyteacup** [upptime/upptime#726]
**Describe the bug**
Most aspects of the Status Page do not update after it's first update. The overall uptime, average response time, and response time graphs never change. Adding new sites to monitor in .upptimerc.yml also never appear on the status page. The stats are correctly reflected in the repo home page. The only page on the status page that does update correctly is the history page, however only the response time plot graph is updated, the other stats are still frozen in time from the first update.

**Comment by Deathbyteacup** [comment]
It seems that raw.githubusercontent.com is so heavily cached, that it's impossible to get up to date data after it first loads.
Changing the capitalisation of your the owner or repo effectively 'cachebusts' this and the page updates. Otherwise, the page will remain static and never change.

**Comment by AnandChowdhary** [comment]
You're right, this was a bug which should be resolved by GitHub soon: https://github.com/orgs/community/discussions/46691#discussioncomment-4948308
